Bifunctor/package.dhall

```diff
@@ -0,0 +1,58 @@
+    let Bifunctor = ./Type
+
+in  let Module
+        :   ∀(p : Type → Type → Type)
+          → ∀ ( bifunctor
+              : { bimap :
+                      ∀(a : Type)
+                    → ∀(b : Type)
+                    → ∀(c : Type)
+                    → ∀(d : Type)
+                    → ∀(f : a → b)
+                    → ∀(g : c → d)
+                    → ∀(pac : p a c)
+                    → p b d
+                }
+              )
+          → { bimap :
+                  ∀(a : Type)
+                → ∀(b : Type)
+                → ∀(c : Type)
+                → ∀(d : Type)
+                → ∀(f : a → b)
+                → ∀(g : c → d)
+                → ∀(pac : p a c)
+                → p b d
+            , first :
+                  ∀(a : Type)
+                → ∀(b : Type)
+                → ∀(c : Type)
+                → ∀(f : a → b)
+                → ∀(pac : p a c)
+                → p b c
+            , second :
+                  ∀(a : Type)
+                → ∀(c : Type)
+                → ∀(d : Type)
+                → ∀(g : c → d)
+                → ∀(pac : p a c)
+                → p a d
+            }
```

Comment:
We have stopped type annotating expressions apart from instances so we can drop the above. We can also drop the `let Module ... in Module` and just keep the record expression.